executors.go

```diff
@@ -393,11 +393,11 @@ func (c *Connection) UpdateColumns(model interface{}, columnNames ...string) err
 
 			cols := columns.Columns{}
 			if len(columnNames) > 0 && tn == sm.TableName() {
-				cols = columns.NewColumnsWithAlias(tn, m.As)
+				cols = columns.NewColumnsWithAlias(tn, m.As, sm.IDField())
 				cols.Add(columnNames...)
 
 			} else {
-				cols = columns.ForStructWithAlias(model, tn, m.As)
```

Comment:
Same here, `model` was the outer model.

Reply:
See https://github.com/gobuffalo/pop/pull/604/files#r513381638

Reply:
See https://github.com/gobuffalo/pop/pull/604#discussion_r514195713